salvo/src/lib/docker/test_docker_image_builder.py

```diff
@@ -13,6 +13,8 @@
 import api.source_pb2 as proto_source
 import api.control_pb2 as proto_control
 
+_default_envoy_image_tag = "envoy/envoy-dev:envoy_tag"
```

Comment:
I must have missed these before, sorry about that. As constants, their names should also be all-caps.

Reply:
I think there are a few pre-existing ones in this file also, but we can address those in a separate PR.

Thank you for updating.